### ISSUE TITLE
Can't build at the current Coq submodule commit

### ISSUE THREAD

**Issue by relrod** [HoTT/HoTT#736]
When running `etc/install_coq.sh`, it makes some progress before failing with:

```
...
cd kernel/byterun/ && \
"ocamlmklib" -oc coqrun coq_fix_code.o coq_memory.o coq_values.o coq_interp.o
OCAMLC stm/proofworkertop.ml
OCAMLC -a -o stm/proofworkertop.cma
OCAMLC stm/tacworkertop.ml
OCAMLC -a -o stm/tacworkertop.cma
OCAMLC stm/queryworkertop.ml
OCAMLC -a -o stm/queryworkertop.cma
COQMKTOP -o bin/coqtop.byte
File "lib/lib.cma(Errors)", line 1:
Warning 31: files lib/lib.cma(Errors) and /usr/lib64/ocaml/compiler-libs/ocamlbytecomp.cma(Errors) both define a module named Errors
File "parsing/parsing.cma(Lexer)", line 1:
Warning 31: files parsing/parsing.cma(Lexer) and /usr/lib64/ocaml/compiler-libs/ocamlcommon.cma(Lexer) both define a module named Lexer
cp bin/coqtop.byte bin/coqtop
COQC -noinit theories/Init/Notations.v
Fatal error: exception Errors.Anomaly(_, _)
Makefile.build:585: recipe for target 'theories/Init/Notations.vo' failed
make[1]: *** [theories/Init/Notations.vo] Error 2
make[1]: Leaving directory '/home/ricky/devel/coq/hott/coq-HoTT'
Makefile:158: recipe for target 'submake' failed
make: *** [submake] Error 2
ricky@t520 98% ~/devel/coq/hott (master)$ 
```

Happy to provide whatever other information is helpful -- I tried building with a newer Coq version but was unable to find a combination that worked and didn't run into build issues somewhere else along the line. Is there a newer Coq commit that I can/should try against?

**Comment by JasonGross** [comment]
Travis is happy, so I'm not sure what's going wrong.  Can you try all of this in a fresh clone of the HoTT repository, and see if you can reproduce it there?  It sounds like you might have managed to get a Coq binary using compiled Ocaml files from multiple different versions of Coq.

**Comment by relrod** [comment]
@JasonGross I believe that output was from a clean clone of the repo, but I will try again later today when I'm not stuck on battery power.

**Comment by JasonGross** [comment]
We now use the released version Coq 8.5pl1
